src/lsst/cm/tools/core/error_code_decisions.yaml

```diff
@@ -136,6 +145,25 @@ pandaErrorCode:
             rescue: 1
             panda: 1
             intensity: 0
+        p_fgcm_380:
+            diagMessage: "Exception KeyError: (380, 40)"
+            function: fgcmBuildStarsTable
+            known: 1
+            ticket: ["DM-37570"]
+            resolved: 0
+            rescue: 0
+            panda: 0
+            intensity: 0
+        keyerror_0:
+            diagMessage: >
+                in run exposure = input_exposures[detector_id].get() KeyError: 0
```

Comment:
Ah, yeah, `in run` was a mistake. 
To preserve the newline I used the multiline yaml character `|` instead of `>` : 
`diagMessage: |
                exposure = input_exposures[detector_id].get()
                KeyError: 0`
I _think_ that is correct